tests/integration/acl.feature

```diff
@@ -196,7 +196,7 @@ Feature: API Access Control Layer
         When I request "/posts"
         Then the guzzle status code should be 200
         And the response is JSON
-        And the "count" property equals "13"
+        And the "count" property equals "12"
```

Comment:
I stared at these tests for a while and ran them manually via postman and am convinced that these updated numbers are now correct. What is happening is that without the new limit these queries pull 21+ results from the db and THEN SearchUsecase has a block of code around line 95 that "remove[s] any entities that cannot be seen" i.e. those that auth fails for the particular user. It just so happens that 1 of the "good" results is the 21st result that is left out by the limited query, so we end up with 1 less in the result. I think this serves to prove that the limiter for unprivileged requests is actually working.

Reply:
Ah right... that's a yucky thing of the current implementation. Makes sense 👍

Reply:
Yeah I debated whether to try to fix it to have php do the limiting after the inaccessible records are removed but didn't want to blow up the status quo at this point. That seems like a future design consideration that likely exists across all the assets.